float64 EE
float64 NN
float64 acc_x
float64 acc_y
float64 acc_z
float64 pitch_
float64 roll_
float64 yaw_
